src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -136,7 +136,11 @@ public enum ErrorCode {
   INVALID_QUERY("Invalid query"),
   DRIVER_TIMEOUT("Driver timeout"),
   DRIVER_CLOSED_CONNECTION("Driver request connection is closed"),
-  NO_NODE_AVAILABLE("No node was available to execute the query");
+  NO_NODE_AVAILABLE("No node was available to execute the query"),
+  NO_INDEX_ERROR(
```

Comment:
Since this is thrown at query time (if I understand it correctly), not creation time, it should probably be worded to indicate that Collection is missing indexes (possibly due to faulty creation), but not that operation failing was CreateCollection. To me latter would be confusing as a user.

Reply:
thanks. make sense!